### ISSUE TITLE
re-position window --win while playing

### ISSUE THREAD

**Issue by OliverGruenberg** [popcornmix/omxplayer#685]
Hi there,

is it possible to re-position the window of a video while it is playing?
Like sending a signal or a position refresh via a simple shell command?

**Comment by popcornmix** [comment]
Yes, it should be possible. Have a look at: https://github.com/popcornmix/omxplayer/pull/149

**Comment by OliverGruenberg** [comment]
thank you, I've tried the samples like this:

dbus-send --print-reply=literal --session --dest=org.mpris.MediaPlayer2.omxplayer /org/mpris/MediaPlayer2 org.mpris.MediaPlayer2.Player.VideoPos objpath:/not/used string:"0 0 50 50"

It results in:
Failed to open connection to "session" message bus: Unable to autolaunch a dbus-daemon without a $DISPLAY for X11

Seems like it only works in a X11 environment although omxplayer itself works fine without?

I'm working on https://github.com/guysoft/FullPageOS, a headless Raspbian in Chromium kiosk mode...

**Comment by popcornmix** [comment]
I believe dbus expects X11, so this may be difficult without X.
It something like this an option?
https://unix.stackexchange.com/questions/105964/launch-a-fake-minimal-x-session-for-pulseaudio-dbus
